### ISSUE TITLE
Investigate accessibility of `RangeInput` 

### ISSUE THREAD

**Issue by dpikt** [LaunchPadLab/lp-components#348]
Looks like there might be some enhancements we can make: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_slider_role

**Comment by chawes13** [comment]
It looks like we're also using a `label` element for the range, which likely should be a span or other element with an appropriate `aria-labelledby` or `aria-describedby` attribute
